guestbook/php-redis/Dockerfile

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM php:5-apache
+FROM BASEIMAGE
```

Comment:
@sebgoa pretty much every other Dockerfile in the k/k, k/contrib and other repos use this standardized pattern. I think adding a comment above way better than maintaining many copies.